tests/nvidiagpu/deploy-gpu-test.go

```diff
@@ -627,7 +627,14 @@ var _ = Describe("GPU", Ordered, Label(tsparams.LabelSuite), func() {
 
 			By("Deploy ClusterPolicy")
 			glog.V(gpuparams.GpuLogLevel).Infof("Creating ClusterPolicy from CSV almExamples")
-			clusterPolicyBuilder := nvidiagpu.NewBuilderFromObjectString(inittools.APIClient, almExamples)
+
+			var clusterPolicyBuilder *nvidiagpu.Builder
+			if nvidiaGPUConfig.ClusterPolicyPatch == "" {
+				clusterPolicyBuilder = nvidiagpu.NewBuilderFromObjectString(inittools.APIClient, almExamples)
+			} else {
+				clusterPolicyBuilder = nvidiagpu.NewBuilderFromObjectStringAndPatch(inittools.APIClient, almExamples, nvidiaGPUConfig.ClusterPolicyPatch)
```

Comment:
Would be good to add a glog message to mention we are using a patched clusterpolicy and not the default one from alm-example.   This helps in troubleshooting.   Also would be more consistent if we move the check for this env var in the BeforeAll() section where we check all other env vars and output the input values from NVIDIAGPU_GPU_CLUSTER_POLICY_PATCH env var in the beginning of the log, for better troubleshooting.